meson.build

```diff
@@ -1740,6 +1740,9 @@ if get_option('libmpv')
     headers = ['libmpv/client.h', 'libmpv/render.h',
                'libmpv/render_gl.h', 'libmpv/stream_cb.h']
     install_headers(headers, subdir: 'mpv')
+
+    libmpv_dep = declare_dependency(link_with: libmpv)
```

Comment:
consider adding comment here w/ explanation

Reply:
Basically cargoculted the libplacebo comment.